extending-valtimo/document/search-fields.md

```diff
@@ -0,0 +1,110 @@
+# Search fields
+
+Search fields add the ability to filter cases. Each search field adds the ability to filter on a specific field.
+This page describes how the search fields can be created, read, updated and deleted.
+
+## Prerequisites
+Only administrators can configure search fields, so the creation, update and deletion of search fields can only be done by administrators.
```

Comment:
The correct role (administrator by default) that allows configurating search fields.

Reply:
The required role is hardcoded so how can it be anything other than administrator

Reply:
Because implementations can make their own security configurer that opens it up for other roles